static/app/views/externalRedirect/index.tsx

```diff
@@ -0,0 +1,92 @@
+import {useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+import dompurify from 'dompurify';
+
+import Panel from 'sentry/components/panels/panel';
+import {t, tct} from 'sentry/locale';
+import {isUrl} from 'sentry/utils';
+
+function ExternalRedirect() {
+  const [count, setCount] = useState<number>(5);
+  const [redirectUrl, setRedirectUrl] = useState<string | null>(null);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    const queryParams = new URLSearchParams(window.location.search);
+    const url = queryParams.get('url');
+
+    if (url && isUrl(url)) {
+      const sanitizedUrl = dompurify.sanitize(url);
```

Comment:
DOMPurify is good when you are setting the HTML to user-provided content, but it doesn't sanitize links. I believe the `isUrl()` function we have is good enough.